### PR TITLE
DO NOT MERGE - Zendesk Connector 1.0

### PR DESCRIPTION
**Pull request by dhirsch1**

### CODE REVIEW

File: zendesk/1.0/antora.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+name: zendesk-connector
+version: '1.0'
+display_version: 1.0 (Mule 4)
+title: Zendesk Connector
+nav:
+- modules/ROOT/nav.adoc
+asciidoc:
+  attributes:
+    page-component-desc: Enables you to connect Mule flows to the Zendesk platform using the Zendesk REST API.
```
Comment (DBCSV) [inline zendesk/1.0/antora.yml:9]: rename to Zendesk's REST Support API please.

File: zendesk/1.0/antora.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    page-connector-type: Connector
+    page-connector-level: Select
+    page-exchange-group-id: org.mule.extension
```
Comment (DBCSV) [inline zendesk/1.0/antora.yml:12]: com.mulesoft.connectors

File: zendesk/1.0/antora.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    page-exchange-asset-id: mule-zendesk-connector
```
Comment (DBCSV) [inline zendesk/1.0/antora.yml:13]: mule4-zendesk-connector

File: zendesk/1.0/antora.yml [hunk 4]
```diff
@@ -0,0 +1,16 @@
+name: zendesk-connector
+version: '1.0'
```
Comment (DBCSV) [inline zendesk/1.0/antora.yml:2]: The version will be 1.0.0

File: zendesk/1.0/antora.yml [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+display_version: 1.0 (Mule 4)
```
Comment (DBCSV) [inline zendesk/1.0/antora.yml:3]: 1.0.0

**Comment by dhirsch1** [comment]
Did not use this PR

**Pull request closed without merging**